src/server/migrations/0.7.0-0.7.1/sql/preferences/alter_preferences_type.sql

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+DO $$ BEGIN
+    ALTER TYPE graph_type ADD VALUE 'map' AFTER 'compare';
```

Comment:
changed lines to reflect add_meter_type.sql

Reply:
Migrations are something we only recently got working correctly, esp. for changing an enum. Thus, I have pushed changes to get this to work as expected. I tested on my machine and the migration worked correctly. Note I did not push the version number change in package.json that is needed as this will be done when the release is made.